src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -865,5 +865,10 @@ public string RoleClaimType
         /// The default is <c>null</c>.
         /// </summary>
         public IEnumerable<string> ValidTypes { get; set; }
+
+        /// <summary>
+        /// Gets or sets the flag that indicates whether to include the <see cref="SecurityToken"/> when the validation fails.
+        /// </summary>
+        public bool IncludeTokenOnFailedValidation { get; set; } = false;
```

Comment:
The properties in this class seem to be in alphabetical order. Consider adding IncludeTokenOnFailedValidation in the same order as others.